packages/ui-buttons/src/ToggleButton/ToggleButtonLocator.js

```diff
@@ -25,8 +25,7 @@
 import { locator } from '@instructure/ui-test-locator'
 import { find, parseQueryArguments } from '@instructure/ui-test-queries'
 
-// eslint-disable-next-line no-restricted-imports
-import { TooltipLocator } from '@instructure/ui-tooltip/es/Tooltip/TooltipLocator'
+import { TooltipLocator } from '@instructure/ui-tooltip'
```

Comment:
With the Node 12+ one can only access methods that are defined in the `exports` field, it could be that these were not working since a while.